### ISSUE TITLE
[FE]:  A creator has multiple options for distributing their hypercert

### ISSUE THREAD

**Issue by ccerv1** [hypercerts-org/hypercerts#280]
### Describe the feature you'd like to request

On the create form, a user can toggle between different options:
 - They can upload or link to an allowlist (csv); total supply = num fractions on the allowlist
 - They can fill out a "lite" allowlist (eg, a split between a small number of contributors)
 - They can set a total supply of fractions and then auto-claim 100%

### Describe the solution you'd like

A form that can be easily integrated into the Create page UI

### Describe alternatives you've considered

Maintaining the status quo (ie, provide allowlist link OR creator gets 100%)

**Comment by marsrobertson** [comment]
Related: https://github.com/hypercerts-org/hypercerts/issues/427

TLDR: textarea with `address` and `fractions`, one pair per line

**Comment by bitbeckers** [comment]
In https://github.com/hypercerts-org/HIPs/discussions/5
